app.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const app = express();
+const healthcheck = require("./controllers/healthcheck.controller");
+const { getTopics } = require("./controllers/topics.controller");
+// const {} = require("./controllers/articles.controller");
+// const {} = require("./controllers/comments.controller");
+// const {} = require("./controllers/users.controller");
+
+app.use(express.json());
+
+//Endpoints:
+
+app.get("/api/healthcheck", healthcheck);
+
+app.get("/api/topics", getTopics);
+
+// Error handling:
+
+app.use((err, request, response, next) => {
+  if (err.status) {
+    response.status(err.status).send({ msg: err.msg });
+  }
+  next(err);
+});
+
+app.use((err, request, response, next) => {
+  response.status(500).send({ message: "internal server error" });
+});
+
```

Comment:
great error handlers! You may want to think about moving these into another file in order to make sure that you don't weigh down the app.js with error handling. 

Also, you should consider what error handler you should use when the endpoint has been written incorrectly- i.e. '/api/wrongendpoint'